Sources/Core/Internal/DisplayLink-macOS.swift

```diff
@@ -0,0 +1,44 @@
+//
```

Comment:
Could you use the same style of header for the new files as the rest of the repo? 😇